emerge-gitclone

```diff
@@ -9,17 +9,36 @@ import sys
 
 import portage
 
-branch = 'master'
+# Find the version of the currently running release.
+release = False
 with open('/usr/share/coreos/release') as release_file:
 	for line in release_file:
 		if line.startswith('COREOS_RELEASE_VERSION='):
-			branch = 'build-' + line.replace('.', '=', 1).split('=', 2)[1]
+			release = line.split('=', 1)[1].strip()
+
+# Attempt to read Git commits from this release's manifest.
+branch = 'master'
+commits = {}
+if release:
```

Comment:
Idiomatically `if release is not None` to avoid confusion in case of third-party classes with falsy values.  Doesn't strictly matter in this case.